src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/AllocationsMigrationService.kt

```diff
@@ -146,6 +148,28 @@ class AllocationsMigrationService(
           null,
         )
       }
+
+  fun toSlots(exclusions: List<AllocationExclusion>): List<Slot> =
```

Comment:
Not plumbed in yet, just wanted to get the conversion from NOMIS to DPS working first.